File: StokTakipFE/src/app/services/serviceClasses/ApiCategoryDefinitionResponse.ts
export interface ApiCategoryDefinitionResponse {
    id: string;
    type: number;
    name: string;
    parentId: string;
    lastUpdate?: Date;
    editedBy?: string;
    deleted: boolean;
    createdBy?: string;
    createdDate?: Date;
}

export class ApiCategoryDefinitionResponse implements ApiCategoryDefinitionResponse {
    id: string;
    type: number;
    name: string;
    parentId: string;
    lastUpdate?: Date;
    editedBy?: string;
    deleted: boolean;
    createdBy?: string;
    createdDate?: Date;

    constructor(o: ApiCategoryDefinitionResponse) {
        if (!o || !o.id) {
            return;
        }
        this.id = o.id;
        this.type = o.type;
        this.name = o.name;
        this.parentId = o.parentId;
        this.lastUpdate = o.lastUpdate;
        this.editedBy = o.editedBy;
        this.deleted = o.deleted;
        this.createdBy = o.createdBy;
        this.createdDate = o.createdDate;
    }
}

File: StokTakip/StokTakip/ApiClasses/ApiCategoryDefinitionResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace StokTakip.ApiClasses
{
    public class ApiCategoryDefinitionResponse
    {
        public Guid id { get; set; }
        public short type { get; set; }
        public string name { get; set; }
        public Guid parentId { get; set; }
        public DateTime? lastUpdate { get; set; }
        public Guid? editedBy { get; set; }
        public bool deleted { get; set; }
        public Guid? createdBy { get; set; }
        public DateTime? createdDate { get; set; }
    }
}
